rosa_tutorials/rosa-mobb-external-dns-for-custom-domain-tutorial.adoc

```diff
@@ -0,0 +1,328 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-external-dns-for-custom-domain-tutorial"]
+= Tutorial: Deploying the external DNS Operator with a custom domain in ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: ROSA-mobb-external-dns-for-custom-domain-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2021-06-10'
+//title: External DNS for ROSA custom domain
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Chris Kang
+//  - Dustin Scott
+//---
+
+Configuring the `Custom Domain` Operator requires a wildcard CNAME DNS record in your Route53 hosted zone. If you do not want to use a wildcard record, you can use the `External DNS` Operator to create individual entries for routes.
+
+This document will guide you through deploying and configuring the External DNS Operator with a custom domain in Red Hat OpenShift Service on AWS (ROSA).
+
+[IMPORTANT]
+====
+The `External DNS` Operator does not support STS yet and uses long lived IAM credentials. This guide will be updated once STS is supported.
+====
+
+[id="rosa-mobb-external-dns-for-custom-domain-prerequisites"]
+== Prerequisites
+
+* A user account with `dedicated-admin` privileges
+* Access to a ROSA cluster with the latest version of the `oc` CLI installed
+* The latest AWS CLI
+* The latest `rosa` CLI
+* Route53 hosted zone
+* A domain
+
+[id="rosa-mobb-external-dns-for-custom-domain-deploy"]
+== Deploy
+
+Deploy the External DNS Operator by completing the following sections.
+
+=== Setup environment
+
+. Set your email and your domain:
++
+[source,terminal]
+----
+$ export EMAIL=<YOUR-EMAIL>
```

Comment:
When I found commands like these `export` commands, I combined all of them into one block. We aren't consistent with these variables, and we didn't do it much in the documentation previously, so whatever makes the most sense works for me.